src/cpp/session/resources/schema/user-prefs-schema.json

```diff
@@ -1144,10 +1144,17 @@
             "default": "",
             "description": "The path to the terminal executable to use."
         },
-        "rsa_key_path": {
+        "ssh_key_path": {
```

Comment:
I wouldn't change an existing preference's name. Yes, it would be more accurate and self-documenting, but if you do this you should also implement migration of the setting from the old value to the new, and I don't think that's worth the effort to code and test. Plus nothing you can do if user goes backwards (downgrades to an older version).

Reply:
ah right, yes good point!